xml/net_dns.xml

```diff
@@ -735,7 +735,7 @@
    right after installation without any problems. Normally, if you already have an Internet connection and entered
    <systemitem class="ipaddress">127.0.0.1</systemitem> as the name server
    address for <systemitem class="domainname">localhost</systemitem> in
-   <filename>/etc/resolv.conf</filename>, you already have a working
```

Comment:
I'd stick to /etc/resolv.conf here, too. Maybe mention to not edit the file by hand?

Reply:
Set back to /etc/resolv.conf.